release-v1.3.2 AccountAgeWitnessServiceTest > testArbitratorSignWitness FAILED

<!--
   SUPPORT REQUESTS: This is for reporting bugs in the Bisq app.
   If you have a support request, please join #support on Bisq's
   Keybase team at https://keybase.io/team/Bisq
-->

### Description

Core's testArbitratorSignWitness fails during cmd line build, therefore jars, start scripts and packages cannot be built (unless tests are not run).

#### Version

Release branch release/v1.3.2

### Steps to reproduce

Linux:  ./gradlew clean build

### Expected behaviour

All tests should pass

### Actual behaviour

Build fails:

    bisq.core.account.witness.AccountAgeWitnessServiceTest > testArbitratorSignWitness FAILED
        java.lang.AssertionError at AccountAgeWitnessServiceTest.java:220
            AssertionError: expected:<0> but was:<2>

The assertion error's wording is a bit misleading.  It should say expected:<2> but was:<0>
after the assert arguments are corrected:   `assertEquals(items.size(), 2);` 
is changed to `assertEquals(2, items.size());`

#### Device or machine

Linux / Ubuntu 18 / OpenJDK 11


#### Additional info

This tests PASSES when run in Intellij.

I have run the cmd line build (w/ tests), and run AccountAgeWitnessServiceTest in IDE several times to make sure this is the case:  fails on cmd line, passes in IDE.

Note that this failing test is the only one of three test cases in AccountAgeWitnessServiceTest that is not ignored.

--- a/core/src/test/java/bisq/core/account/witness/AccountAgeWitnessServiceTest.java
+++ b/core/src/test/java/bisq/core/account/witness/AccountAgeWitnessServiceTest.java
@@ -62,6 +62,7 @@ import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 
+import static bisq.core.payment.payload.PaymentMethod.getPaymentMethodById;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
@@ -216,8 +217,8 @@ public class AccountAgeWitnessServiceTest {
         when(contract.getBuyerPaymentAccountPayload()).thenReturn(buyerPaymentAccountPayload);
         when(contract.getSellerPaymentAccountPayload()).thenReturn(sellerPaymentAccountPayload);
         when(contract.getOfferPayload()).thenReturn(mock(OfferPayload.class));
-        List<TraderDataItem> items = service.getTraderPaymentAccounts(now, PaymentMethod.SEPA, disputes);
-        assertEquals(items.size(), 2);
+        List<TraderDataItem> items = service.getTraderPaymentAccounts(now, getPaymentMethodById(PaymentMethod.SEPA_ID), disputes);
+        assertEquals(2, items.size());
 
         // Setup a mocked arbitrator key
         ECKey arbitratorKey = mock(ECKey.class);

